4-perftools/test/expected-usage

```diff
@@ -0,0 +1,21 @@
+This is a devtoolset-4-perftools base image.  It contains the dts-4
+systemtap, dyninst (+devel), elfutils (+devel), valgrind packages.
+These are usually intended to monitor the host or other containers,
+so this image should normally be launched in a privileged container.
+
+Sample invocation:
+
+   % atomic run --spc devtoolset-4-perftools-rhel7 /bin/bash
+or % docker run -ti --privileged --ipc=host --net=host --pid=host [...]             devtoolset-4-perftools-rhel7 /bin/bash
+
+
+For using systemtap, also install a kernel, kernel-devel,
+plus possibly kernel-debuginfo, all matching the container
+host, within the container:
+
+   % atomic run devtoolset-4-perftools-rhel7 /bin/bash
+   # yum -y install kernel kernel-devel kernel-debuginfo
+
+and save that container to a reusable image:
+
+   % docker commit  devtoolset-4-perftools-rhel7-3.10.0-327.10.1.el7.x86_64
```

Comment:
Good catch, indeed the test fails on my Fedora laptop. Added a commit to address this issue, and tested on my laptop.